ImageInLib/ImageInLib/src/segmentation2D_lagrangean.c

```diff
@@ -75,25 +73,31 @@ bool lagrangeanExplicit2DCurveSegmentation(Image_Data2D inputImage2D, const Lagr
     }
 
     //get edge detector
-    
     for (size_t i = 0; i < dataDimension; i++)
     {
-        edge_detector[i] = edgeDetector(abs_val_grad[i], edge_detector_coef);
+        edge_detector[i] = edgeDetector(abs_val_grad[i], pSegmentationParams->edgeCoef);
     }
 
+    unsigned char saveEdge[] = "C:/Users/Konan Allaly/Documents/Tests/Curves/Output/edge_detector.raw";
```

Comment:
please remove or at least comment out the absolute path

Reply:
Done